codersdk/agentsdk/agentsdk.go

```diff
@@ -696,8 +696,9 @@ const (
 )
 
 type ReinitializationEvent struct {
-	Message string                 `json:"message"`
-	Reason  ReinitializationReason `json:"reason"`
+	WorkspaceID uuid.UUID
+	UserID      uuid.UUID
```

Comment:
This bleeds the abstraction a bit because there's nothing prebuilds-specific in this func. The workspace's agent is what reinitializes, and the user _should_ be irrelevant here. Do we need this?